sdk/iot/Azure.Iot.Hub.Service/tests/E2eTestBase.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    /// <summary>
+    /// This class will initialize all the settings and create and instance of the IoT Hub service client.
+    /// </summary>
+    [Parallelizable(ParallelScope.Self)]
+    public abstract class E2eTestBase : RecordedTestBase<IotHubServiceTestEnvironment>
+    {
+        public E2eTestBase(bool isAsync)
+         : base(isAsync, TestSettings.Instance.TestMode)
+        {
+        }
+
+        public E2eTestBase(bool isAsync, RecordedTestMode testMode)
```

Comment:
This is not needed anymore. Will remove from ADT as well. You can remove it here.

Reply:
I'll sync up with you offline on this.

Reply:
Tracked here: https://msazure.visualstudio.com/One/_workitems/edit/7112055